src/StockportContentApi/Model/Link.cs

```diff
@@ -1,4 +1,4 @@
 ﻿namespace StockportContentApi.Model
 {
-    public record Link(string Url, string Text);
+    public record Link(string Url, string Text, bool External);
```

Comment:
prefer to have 'is' append' at the start of a bool variable so it is obvious. eg isExternal  and it will look nicer in if statement if (isExternal){}